LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1346,6 +1346,7 @@ webkit.org/b/241265 [ Debug ] imported/w3c/web-platform-tests/html/rendering/rep
 
 # NOTIFICATION_EVENT is Ventura+
 [ Ventura+ ] http/tests/workers/service/shownotification-allowed-document.html [ Pass ]
+[ Ventura+ ] http/tests/workers/service/shownotification-allowed.html [ Pass ]
```

Comment:
Ventura is the oldest supported macOS.

Reply:
I removed Ventura+ from these expectations.